frontend/src/components/abfragen/AbfrageComponent.vue

```diff
@@ -1,5 +1,63 @@
 <template>
   <div>
+    <div v-if="!isSchnellesErledigtStepper()">
```

Comment:
Ich denke, dass es besser wäre den Stepper für die Statusdarstellung in einen eigenen Stepper zu hinterlegen.

Es handelt such um eine eigenständige Komponente welche eine in sich geschlossene Logik besitzt.